frontend/packages/console-plugin-sdk/src/typings/pages.ts

```diff
@@ -10,6 +10,21 @@ namespace ExtensionProperties {
     loader: LazyLoader<T>;
   }
 
+  /** To add an additonal page to public components(ex: PVs, PVCs) via plugins */
+  export type AdditionalPage = ResourcePage<{
```

Comment:
Call this `ResourceTab` or something like it. `AdditionalPage` doesn't convery what this is.